### PR TITLE
feat: blog release NFTUp

### PR DESCRIPTION
**Pull request by jchris**
link to NFTUp download and screenshots

### CODE REVIEW

**Comment by jchris** [comment]
Screenshot 2022-03-31 at 15 10 59
Screenshot 2022-03-31 at 15 10 59
Screenshot 2022-03-31 at 15 14 17
Screenshot 2022-03-31 at 15 19 49

File: packages/website/posts/2022-04-04-announcing-nftup [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+People have been asking for an easy way to drag and drop large directories onto IPFS and NFT.Storage — today we deliver.
+
+Click to download the latest release, follow the link that matches your computer, or [browse the repo's GitHub releases page](https://github.com/nftstorage/nftup/releases):
+
+* [M1 Mac](https://github.com/nftstorage/nftup/releases/download/v1.0.0-alpha.6/NFT-UP-1.0.0-alpha.6-arm64-mac.zip)
+* [Intel Mac](https://github.com/nftstorage/nftup/releases/download/v1.0.0-alpha.6/NFT-UP-1.0.0-alpha.6-mac.zip)
+* [Windows](https://github.com/nftstorage/nftup/releases/download/v1.0.0-alpha.6/NFT-UP-Setup-1.0.0-alpha.6.exe)
+* [Linux x86](https://github.com/nftstorage/nftup/releases/download/v1.0.0-alpha.6/NFT-UP-1.0.0-alpha.6.dmg)
+* [Linux ARM](https://github.com/nftstorage/nftup/releases/download/v1.0.0-alpha.6/NFT-UP-1.0.0-alpha.6-arm64.dmg)
```
Comment (alanshaw) [inline packages/website/posts/2022-04-04-announcing-nftup:21]: FYI - linux and mac are mixed up

Comment (jchris) [reply packages/website/posts/2022-04-04-announcing-nftup:21]: thanks

**Comment by jchris** [comment]
Screenshot 2022-03-31 at 15 09 35

**Comment by jchris** [comment]
Screen Shot 2022-04-04 at 11 52 41 AM

**Comment by jchris** [comment]
Screen Shot 2022-04-04 at 11 54 35 AM

**Pull request merged**